package.json

```diff
@@ -1,63 +1,102 @@
 {
-    "name": "react-base-app",
-    "private": true,
-    "version": "0.0.0",
+    "name": "alert-hub",
+    "version": "7.1.2",
     "type": "module",
+    "private": true,
     "scripts": {
+        "initialize:type": "mkdir -p generated/",
+        "prestart": "yarn initialize:type",
+        "pretypecheck": "yarn initialize:type",
+        "prelint:js": "yarn initialize:type",
+        "prelint:unused": "yarn initialize:type",
+        "prebuild": "yarn initialize:type",
         "start": "vite",
-        "dev": "vite",
         "build": "vite build",
-        "preview": "vite preview",
-        "check-unused": "unimported",
-        "test": "vitest",
-        "coverage": "vitest run --coverage",
+        "generate:type": "yarn generate",
         "typecheck": "tsc",
-        "lint": "eslint src",
-        "css-lint": "stylelint ./src/**/*.css"
+        "lint:js": "eslint src",
+        "lint:css": "stylelint \"./src/**/*.css\"",
+        "lint:unused": "unimported",
+        "lint:translation": "node ./scripts/translator.js",
+        "lint": "yarn lint:js && yarn lint:css && yarn lint:unused && yarn lint:translation",
+        "test": "vitest",
+        "test:coverage": "vitest run --coverage",
+        "postinstall": "patch-package"
     },
     "dependencies": {
-        "@togglecorp/fujs": "^2.0.0",
+        "@ifrc-go/icons": "^1.3.1",
+        "@mapbox/mapbox-gl-draw": "^1.2.0",
+        "@sentry/react": "^7.81.1",
+        "@tinymce/tinymce-react": "^4.3.0",
+        "@togglecorp/fujs": "^2.1.1",
+        "@togglecorp/re-map": "^0.2.0-beta-6",
+        "@togglecorp/toggle-form": "^2.0.4",
+        "@togglecorp/toggle-request": "^1.0.0-beta.2",
+        "@turf/bbox": "^6.5.0",
+        "@turf/buffer": "^6.5.0",
+        "exceljs": "^4.3.0",
+        "file-saver": "^2.0.5",
+        "html-to-image": "^1.11.11",
```

Comment:
I don't think we need them. Any reasons to add these dependencies.  Before adding dependencies, it's crucial to evaluate whether they align with the project's requirements. Let's refrain from including dependencies preemptively.